### DISCUSSION TITLE
two instances and two different versions?

### DISCUSSION THREAD

**Discussion by osmely** [pockethost/pockethost#159] [Q&A] [answered]
Hello, I have two instances (projects) and I see that one of them remains at version 0.10.0 and the other is automatically updated. Should I do something to keep both of them up to date?

**Comment by benallfree** [comment]
PocketHost will currently will only upgrade patches. It will not cross minor versions because PocletBase is under such active development. 

New instances are created locked to the latest minor version and patch, but the patch can be automatically updated. 

If you tell me your instance name, I can set it to v0.11.0 for you :)

**Comment by osmely** [comment]
It's weird, I thought I saw that one of my instances went from 0.10.0 to 0.10.4 and then to 0.11.0 automatically and the other one didn't. Or maybe I dreamed it haha. My instances are called "profiles" and "profiles-dev".

**Comment by benallfree** [comment, marked as answer]
@osmely Both are now set to `~0.11.0` 👍
